manifest.json

```diff
@@ -10,12 +10,12 @@
     "url": "https://www.mediawiki.org/",
     "license": "GPL-2.0-or-later",
     "maintainer": {
-        "name": "Antoine",
-        "email": "antoine@miaou.org",
-        "url": "https://miaou.org"
+        "name": "decentral1se",
+        "email": "lukewm@riseup.net",
+        "url": "https://decentral1se"
     },
     "requirements": {
-        "yunohost": ">= 3.4"
+        "yunohost": ">= 3.6.4"
```

Comment:
For me 3.5 is enough since the "latest" helper you used is ynh_script_progression introduced in 3.5, did I miss some more recent helper you use?